### ISSUE TITLE
[BUG] Delombok removes "super();"

### ISSUE THREAD

**Issue by lars-sh** [rzwitserloot/lombok#2023]
**Describe the bug**
To me it seems delombok removes "super();" calls for classes with at least one lombok annotation.

**To Reproduce**
Lombok sources:
```
public class First {
	public First() {
		System.out.println("First");
	}
}
```
```
import lombok.Getter;

@Getter
public class Second extends First {
	public Second() {
		super();
		System.out.println("Second");
	}
}
```

Delomboked sources (only class "Second"):
```
public class Second extends First {
	public Second() {
		System.out.println("Second");
	}
}
```

**Expected behavior**
Delomboked sources (only class "Second"):
```
public class Second extends First {
	public Second() {
		super();
		System.out.println("Second");
	}
}
```

**Version info:**
 - Lombok: 1.18.4
 - Platform: org.projectlombok:lombok-maven-plugin:1.18.4.0

**Additional context**
As those calls are done implicitly when missing, this is not a programmatic issue.

Though this
- might be the expected behavior for most developers,
- should bring delombok closer to the original sources,
- allow using analytics (such as PMD) without additional suppressions.

I had a quick look at the delombok sources and to me it seems, simply removing the block https://github.com/rzwitserloot/lombok/blob/2da5d27e9b94864959caf1fe5064adbe0c318196/src/delombok/lombok/delombok/PrettyPrinter.java#L255-L263 should accomplish this. Hope it helps.

**Comment by rspilker** [comment]
At the moment, we don't know if the super call was actually in the source file, or if it was added during compilation. We know that the node is present in the AST, but not where it comes from.

We chose to not print it, because inserting unnecessary super calls would also be confusing.

I'll have a quick look to see if there happens to be an easy way to determine if the statement was in the original source file, but if it is too complicated, we're going to leave it as is.
